src/MICore/CommandFactories/gdb.cs

```diff
@@ -225,5 +225,38 @@ private static string TypeBySize(uint size)
         }
 
         public override bool SupportsDataBreakpoints { get { return true; } }
+
+        public override async Task<TargetArchitecture> GetTargetArchitecture()
+        {
+            string cmd = "show architecture";
```

Comment:
Just to make sure, this gives the architecture of GDB, or the target? One potential issue - what if there was a GDB that supported multiple target architectures?